src/t2iapi/metric/metric_responses.proto

```diff
@@ -15,13 +15,15 @@ option java_package = "com.draeger.medical.t2iapi.metric";
 option java_outer_classname = "MetricResponses";
 
 /*
-This message is used as a response for the CalibrateMetric manipulation. It returns a list of NumericMetricValue/@Value
-or SampleArrayValue/@Samples as strings measured during the calibration. An empty list is returned if no values
-can be provided.
+This message is used as a response for the CalibrateMetric manipulation.
+It returns a list of pm:MetricValue/@Value or pm:MetricValue/@Samples as strings measured during the calibration.
+For further comparison of the value(s) it is required that each returned value has the same unit and floating point
+resolution as defined in the respective pm:AbstractMetricDescriptor.
```

Comment:
I don't think that the descriptor defines a floating point resolution, `@Resolution` is essentially the measurement resolution. What is the purpose of this constraint? Do you want the String to match the value that would be transmitted via SDC, so basically ensure that it's `0.01` if that would be transmitted via SDC and not `0.010`?

Reply:
yes, the String values, which are returned from the manipulation shall be comparable with the values which are delivered over SDC and thus have nothing to do with the @Resolution, i guess this part might be confusing for the user

Reply:
reworked